source/customizations.rst

```diff
@@ -1279,6 +1311,27 @@ To enable this feature ensure that ``rclone`` is installed on the same machine
 that Open OnDemand is installed. You also have to enable the feature through
 the :ref:`configuration entry for enabling remote filesystems <remote_files_enabled>`.
 
+Cancel Interactive Sessions
+---------------------------
+
+We can now cancel an interactive session from the session panel without deleting the session card.
+This functionality will allow users to remove the job from the scheduler and keep the information in the OnDemand interface.
+
+This feature is disabled behind a feature toggle. To enable it, set the configuration property ``cancel_session_enabled: true``.
+For more information on how to configure properties, see :ref:`configuration documentation <configuration_properties>`.
+
+.. warning::
+  *Cancel Interactive Sessions* is an experimental feature.
+  It is fully functional, but it could be removed in future OnDemand releases depending on feedback from the community.
```

Comment:
IDK if we want/need the warning. It's an opt-in feature, if they don't like it they don't have to use it.

Reply:
OK - will remove.

Reply:
fixed